app/views/wishlist/index.html.haml

```diff
@@ -1 +1,14 @@
 %h1 Wishlist
+
+/ someone else should make this look pretty
+%table
+  %thead
+    %th Course
+    %th Actions
+  %tbody
+    - @wishlist.each do |item|
+      %tr
+        %td= link_to item.course, item.course
+        %td
+          = render partial: 'wishlist/add_remove_link', locals: { course: item.course }
+          = render partial: 'wishlist/set_notify_link', locals: { item: item }
```

Comment:
Now that we have the helpers, it's probably simpler to just call the methods here and not bother with the partials. If we do want a partial, we should have one partial for each wishlist item and then just use collection rendering.